open-company/open-company-web
# Readonly percentage

card: https://trello.com/c/8Gyoo7tg

test:
- edit compensation
- switch percentage/currency radio button
- you should always edit dollars
